account_banking_reconciliation/views/account_banking_reconciliation.xml

```diff
@@ -0,0 +1,281 @@
+        <field name="arch" type="xml">
+            <form string="Bank Account Reconciliation Statement" version="7.0">
+                <header>
+                    <button name="action_cancel" type="object"
+                            states="draft,to_be_reviewed"
+                            string="Cancel" icon="fa-ban"/>
+                    <button name="action_review" type="object" states="draft"
+                            string="Ready for Review" icon="fa-forward"/>
+                    <button name="action_process" type="object"
+                            states="to_be_reviewed"
+                            string="Process"/>
+                    <button name="action_cancel_draft" states="cancel,done"
+                            string="Set to Draft" type="object"/>
+                    <button name='refresh_record' string='Refresh'
+                            confirm="Current edits in statement will be lost. Do you want to refresh?"
+                            states="draft" type='object'/>
+                    <field name="state" widget="statusbar" nolabel="1"/>
+                </header>
+                <sheet>
+                    <group col="4" colspan="4">
+                        <field name="account_id"
+                               placeholder="Enter Account Name"/>
+                        <field name="name" placeholder="Enter Name"/>
+                        <field name="ending_date"
+                               placeholder="Enter ending date"/>
+                        <field name="starting_balance"
+                               placeholder="Enter Starting Balance"/>
+                        <field name="ending_balance"
+                               placeholder="Enter Ending Balance"/>
+                        <field name="last_ending_date"
+                               placeholder="Last Statement Date"
+                               readonly="1"/>
+                        <field name="suppress_ending_date_filter"/>
+                        <field name="company_id"
+                               groups="base.group_multi_company"
+                               placeholder="Enter Company Name"/>
+                    </group>
+                    <notebook colspan="5">
+                        <page string="Journal Items">
+                            <label for='account_label'
+                                   string='Deposits, Credits, and Interest'/>
+                            <field colspan="4" mode="tree"
+                                   name="debit_move_line_ids"
+                                   nolabel="1" widget="one2many_list"
+                                   height="300">
+                                <form string="Deposits, Credits, and Interest"
+                                      version="7.0">
+                                    <field name="cleared_bank_account"/>
+                                    <field name="date" readonly="1"/>
```

Comment:
maybe it'd be nice to allow to create manual adjusting journal entries here to advisers, not sure thought